### DISCUSSION TITLE
Help Out !! Please

### DISCUSSION THREAD

**Discussion by TahhaR** [thliebig/openEMS-Project#80] [General]
it stops just after the first timestep !! i get the excitation wich is good but i need also the data to plot my S11 files. i've tried to make very long timesteps and also to change the EndCriteria of 30 dB but it seems to be the same, and OpenEMS stops after the first iterration.
Thanks in Advance for your suggestions 
![HilpMii](https://github.com/thliebig/openEMS-Project/assets/61636792/c224ccf8-71ca-4319-86f5-dddd5d4c3b28)

**Comment by thliebig** [comment]
You have openEMS set to max. 30k timesteps.... These all get done, but are not even close to enough... Just set this to a very high number and only rely on the end criteria...

**Reply by thliebig** [reply]
Please also note that you have lots of unused primitives... Check your structure if everything is meshed properly...

**Reply by thliebig** [reply]
I also just notice that your timestep is way way too small. The nyquist rate is through the roof... You surely have many issues with your mesh/setup...
